x/evm/keeper/keeper.go

```diff
@@ -164,9 +164,9 @@ func (k Keeper) GetAccountStorage(ctx sdk.Context, address common.Address) (type
 
 // GetChainConfig gets block height from block consensus hash
 func (k Keeper) GetChainConfig(ctx sdk.Context) (types.ChainConfig, bool) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixChainConfig)
+	store := ctx.KVStore(k.storeKey)
```

Comment:
do we not need to use type `prefix.Store` anymore?

Reply:
we are not using one here because it would fail if you use `store.Get()` with empty bytes. Previously we could bc it only checked that the key was not `nil`